### DISCUSSION TITLE
In-Network Capitation Example Possible Duplication

### DISCUSSION THREAD

**Discussion by mmorri05** [CMSgov/price-transparency-guide#123] [Q&A] [answered]
Is the "tin" array unnecessarily duplicated in the [In-Network Capitation example](https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-capitation-sample.json) for both the $1400.00 and $2,000.00 negotiated rates?

I'd think all eight "npi" iterations should be under a single "tin".

Or...  Was the intent to have service codes "01" and "02" in BOTH "tin" arrays?

I assume it may be a copy/paste error from one of the other examples where there were different service codes.

**Comment by shaselton-usds** [comment, marked as answer]
Yeah, it was a copy and paste bug -- it has been fixed.
